src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator starts work");

```

Comment:
The message 'Excavator starts work' could be more informative. It should indicate that it is an Excavator that is working. Consider specifying the action the Excavator is performing, for example: 'Excavator is digging.'